src/handlers/issue/generate-nft-mint-signature.ts

```diff
@@ -0,0 +1,127 @@
+import { BigNumber, BigNumberish, ethers, utils } from "ethers";
+import { getPayoutConfigByNetworkId } from "../../helpers/payout";
+import { MaxUint256 } from "@uniswap/permit2-sdk";
+import { keccak256, toUtf8Bytes } from "ethers/lib/utils";
+
+const NFT_MINTER_PRIVATE_KEY = process.env.NFT_MINTER_PRIVATE_KEY as string;
+const NFT_CONTRACT_ADDRESS = process.env.NFT_CONTRACT_ADDRESS as string;
+const NFT_CONTRACT_CHAIN_ID = parseInt(process.env.NFT_CONTRACT_CHAIN_ID as string);
+const SIGNING_DOMAIN_NAME = "NftReward-Domain";
```

Comment:
Should we use a more deliberate string? Why did you choose this?

Reply:
this was hardcoded when the [contract](https://github.com/ubiquity/nft-rewards) was deployed

Reply:
Seems odd to me that we have to pass this in as a parameter. @rndquu rfc

Reply:
it's part of [EIP712](https://eips.ethereum.org/EIPS/eip-712) spec

Reply:
> Protocol designers only need to include the fields that make sense for their signing domain. Unused fields are left out of the struct type.

Reply:
Setting `SIGNING_DOMAIN_NAME` (i.e. contract domain name) and contract version are [required](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/61117c4db8497ba489d5e1e127565a011ed6907a/contracts/utils/cryptography/EIP712.sol#L66) by openzeppelin's implementation of [EIP-712](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/cryptography/EIP712.sol)